src/lib/pubkey/mce/code_based_key_gen.cpp

```diff
@@ -10,25 +10,139 @@
  */
 
 #include <botan/internal/code_based_key_gen.h>
-#include <botan/code_based_util.h>
-#include <botan/gf2m_rootfind_dcmp.h>
-#include <botan/internal/binary_matrix.h>
+#include <botan/internal/code_based_util.h>
 #include <botan/loadstor.h>
-#include <botan/polyn_gf2m.h>
 
 namespace Botan {
 
 namespace {
 
+#define BITS_PER_U32 (8 * sizeof (u32bit))
```

Comment:
I don't think we should do that. sizeof gives you the number of bytes, but a byte *may consist of 8 or more bits*. See also http://en.cppreference.com/w/cpp/language/sizeof